dotCMS/src/main/java/com/dotcms/rest/api/v1/system/AppContextInitResource.java

```diff
@@ -71,8 +71,27 @@ public AppContextInitResource(AppConfigurationHelper helper) {
 	@Produces({ MediaType.APPLICATION_JSON, "application/javascript" })
 	public final Response list(@Context final HttpServletRequest request) {
 		try {
-			final Object configData = this.helper.getConfigurationData(request);
+			// If the request is from a Kubernetes probe (User-Agent starts with kube-probe),
+			// respond with a 200 OK and skip the body to reduce unnecessary payloads.
+			//
+			// This is a temporary solution to efficiently handle Kubernetes probes
+			// without introducing additional endpoints or modifying the deployment configuration.
+			// Probes like liveness and readiness do not require the complete configuration payload
+			// returned by this endpoint. Instead, a lightweight response is sufficient.
+			//
+			// NOTE: Once a dedicated endpoint for probes is implemented, this logic
+			// should be reviewed and potentially removed.
+			// Retrieve the User-Agent header from the incoming request
+			String userAgentHeader = "User-Agent";
+			String kubeProbePrefix = "kube-probe";
+			String userAgent = request.getHeader(userAgentHeader);
+
+			if (userAgent != null && userAgent.startsWith(kubeProbePrefix)) {
```

Comment:
This is returing OK for the probe regardless of any other logic.   The intention is to just return the body being returned and not change the logic so this check needs to occur after we get the configurationData,  if the method call throws an exeception a failure response with the exeption being handled by Jersey as before,   we just do not want to actually return the new ResponseEntityView(configMap) in the reponse when we are accessed from a probe